framework/Samplers/MCMC/MCMC.py

```diff
@@ -131,14 +133,17 @@ def __init__(self):
     self._initialValues = {} # dict stores the user provided initial values, i.e. {var: val}
     self._updateValues = {} # dict stores input variables values for the current MCMC iteration, i.e. {var:val}
     self._proposal = {} # dict stores the proposal distributions for input variables, i.e. {var:dist}
+    self._priorFuns = {} # dict stores the prior functions for input variables, i.e. {var:fun}
     self._burnIn = 0      # integers indicate how many samples will be discarded
     self._likelihood = None # stores the output from the likelihood
     self._logLikelihood = False # True if the user provided likelihood is in log format
     self._availProposal = {'normal': Distributions.Normal(0.0, 1.0),
                            'uniform': Distributions.Uniform(-1.0, 1.0)} # available proposal distributions
     self._acceptDist = Distributions.Uniform(0.0, 1.0) # uniform distribution for accept/rejection purpose
+    self.toBeCalibrated = {} # parameters that will be calibrated
     # assembler objects
-    self.addAssemblerObject('proposal', '-n', True)
+    self.addAssemblerObject('proposal', '-n')
```

Comment:
so these to are mutually exclusive? i guess?

Reply:
updated